### ISSUE TITLE
query hash columns are misordered

### ISSUE THREAD

**Issue by joshelser** [sundeck-io/OpsCenter#263]
```
our account_usage.query_history has
hash, hash_version, param_hash, param_hash_version.
materialized table has
param_hash, hash, param_hash_version, hash_version (but with correct column data types)
reporting views have
param_hash, hash, param_hash_version, hash_version (with wrong data types)
```

the reporting.enriched_query_history view has jumbled data types assigned to the 4 new columns from Snowflake. It appears that we re-order the columns on the materialized table (while retaining the correct data types) and then combined the materialized table ordering to column names but the original ordering of datatypes from account_usage.query_history to the final reporting.enriched_query_history view.

**Comment by joshelser** [comment]
```
{
  "alter_statement": "ALTER TABLE \"INTERNAL_REPORTING_MV\".\"QUERY_HISTORY_COMPLETE_AND_DAILY_INCOMPLETE\" ADD \"QUERY_HASH_VERSION\" NUMBER, \"QUERY_PARAMETERIZED_HASH\" TEXT, \"QUERY_PARAMETERIZED_HASH_VERSION\" NUMBER, \"QUERY_HASH\" TEXT"
}
```
```
{
  "alter_statement": "ALTER TABLE \"INTERNAL_REPORTING_MV\".\"QUERY_HISTORY_COMPLETE_AND_DAILY\" ADD \"QUERY_PARAMETERIZED_HASH\" TEXT, \"QUERY_HASH\" TEXT, \"QUERY_PARAMETERIZED_HASH_VERSION\" NUMBER, \"QUERY_HASH_VERSION\" NUMBER"
}
```
tracing ftw

**Comment by joshelser** [comment]
Resolved by #265
